mmv1/third_party/terraform/.teamcity/components/builds/build_triggers.kt

```diff
@@ -27,7 +27,7 @@ fun Triggers.runNightly(config: NightlyTriggerConfiguration) {
 
     schedule{
         enabled = config.nightlyTestsEnabled
-        branchFilter = "+:" + config.branch // returns "+:/refs/heads/main" if default
+        branchFilter = "+:refs/heads/nightly-test"
```

Comment:
I think the way to approach this would be to revert this change and instead update the value of `DefaultBranchName` in `constants.kt`

We need to allow the incoming NightlyTriggerConfiguration object to control the branch used when triggering builds as this is what allows us to test feature branches, e.g. [here we set the branch value](https://github.com/GoogleCloudPlatform/magic-modules/blob/714237ecee293e88dfb323a449bd3b2de470efc7/mmv1/third_party/terraform/.teamcity/components/projects/feature_branches/FEATURE-BRANCH-major-release-6.0.0.kt#L73) to enable testing for the 6.0.0 release

Reply:
done, the change is also reflected in my fork of the TFGCP so that we can confirm it still works as expected in the next nightly-test build run: https://github.com/BBBmau/terraform-provider-google/commit/63543ddb0c0248356a3617f5c312a5ddebf13e45